### PR TITLE
Adding monitoring annotation spec

### PR DESCRIPTION
**Pull request by tomahawk28**
Follow https://github.com/giantswarm/giantswarm/pull/7705

### CODE REVIEW

File: kubernetes/annotations_and_labels.md
Comment (tomahawk28) [inline kubernetes/annotations_and_labels.md]: Only those 4 lines are actual changes. Adding 3 additional monitoring annotation spec.

File: kubernetes/annotations_and_labels.md
Comment (kopiczko) [inline kubernetes/annotations_and_labels.md]: Let's please not change formatting in the PR introducing new labels.

Comment (tomahawk28) [reply kubernetes/annotations_and_labels.md]: Reverted, and adding only specs now.

File: kubernetes/annotations_and_labels.md
Comment (kopiczko) [inline kubernetes/annotations_and_labels.md]:
```suggestion
- `giantswarm.io/monitoring` - `true|false', indicating monitoring feature on/off on the service.
```

File: kubernetes/annotations_and_labels.md
Comment (kopiczko) [inline kubernetes/annotations_and_labels.md]:
```suggestion
- `giantswarm.io/monitoring-path` - value should indicate monitoring URL path.
```

File: kubernetes/annotations_and_labels.md
Comment (kopiczko) [inline kubernetes/annotations_and_labels.md]:
```suggestion
- `giantswarm.io/monitoring-port` - value should indicate monitoring port on the specific path.
```

**Review by kopiczko — APPROVED** [review]

File: kubernetes/annotations_and_labels.md
Comment (tomahawk28) [reply kubernetes/annotations_and_labels.md]: Thanks :)

**Review by rossf7 — COMMENTED** [review]
@tomahawk28 This LGTM but the convention is to wait for at least 3 approvals on `fmt`.

This is because the changes here impact lots of projects. Please do that on future fmt PRs.

**Comment by tomahawk28** [comment]
👍 but it makes sense to update 3 approval to merge right now? 
image

**Comment by rossf7** [comment]
@tomahawk28 I like it but I posted in #dev to get more opinions.

**Pull request merged**
